src/Collection.php

```diff
@@ -354,16 +354,13 @@ public function duplicate(?callable $comparatorCallback = null, ?callable $acces
     }
 
     /**
-     * Create a new instance with no items.
-     *
-     * @template NewTKey
-     * @template NewT
+     * @pure
      *
-     * @return self<NewTKey, NewT>
+     * @return self<mixed, mixed>
```

Comment:
these template types cannot be inferred from anywhere, so the better option here is to just use `mixed`

Reply:
Fair enough!